src/config.rs

```diff
@@ -28,6 +28,7 @@ pub enum OutputFormat {
     /// Minimal output, usually just the result, such as the MSRV or whether verify succeeded or failed
     Minimal,
     /// No output -- meant to be used for debugging and testing
+    #[value(hide = true)]
```

Comment:
What happens when `hide = true`? Will it not show up in the help (i.e. hidden), or be disabled as a value altogether? 

This value would previously only be set when the `--no-user-output` was given; if it is hidden, not disabled, would it be possible to disable it, so it's not available as `--user-output none`.

Reply:
This setting will only hide it from help and shell completions ([docs](https://docs.rs/clap/latest/clap/builder/struct.PossibleValue.html#method.hide)). That means it's not visible, but still valid to do `--user-output none`.

I've looked around and didn't find any setting that would fully remove this variant from the `ValueEnum` implementation. Would have hoped for something like `#[value(skip)]`, but there isn't.

Currently, the only way seems to be to implement `ValueEnum` by hand. But that's rather easy, so I'll go with that, to keep the original behavior.

Reply:
Oh, I was wrong, there is actually `#[value(skip)]`, but it's not mentioned anywhere in the docs. It's only mentioned in regard to subcommands.

I pushed an update right now, so it'll not be part of the valid arguments anymore. I verified that it's actually excluded, by utilizing `cargo-expand`. Just to make sure, as it's not mentioned in the docs.